google/cloud/storage/client.h

```diff
@@ -2591,12 +2593,8 @@ class Client {
    * which is a POST request to GCS.
    *
    * @param document the policy document.
-   *
-   * @par Helper Functions
-   *
-   * The following functions create a `PolicyDocumentCondition` with less
-   * opportunities for typos: `StartsWith()`, `ExactMatchObject()`,
-   * `ExactMatch()`, `ContentLengthRange()`.
```

Comment:
Why is this comment deleted?

Reply:
Argh... I thought these were the options, my bad, I will send a PR to restore them.